app/App.tsx

```diff
@@ -56,7 +55,7 @@ const {
 const outerContainerId = 'outer-container';
 const pageWrapId = 'page-wrap';
 
-export const App = () => {
+export function App() {
```

Comment:
`lint:fix` automatically changed all of these arrow function declarations, resulting in quite a few changed files.